addons/uom/models/uom_uom.py

```diff
@@ -75,6 +79,7 @@ def _unprotected_uom_xml_ids(self):
         default='reference', required=True)
     ratio = fields.Float('Combined Ratio', compute='_compute_ratio', inverse='_set_ratio', store=False)
     color = fields.Integer('Color', compute='_compute_color')
+    labellable = fields.Boolean('labellable', default=lambda self: self._default_labellable(), help="Print 1 product label per unit transferred")
```

Comment:
```suggestion
    allow_label = fields.Boolean('Labelable', default=lambda self: self._default_labellable(), help="Print 1 product label per unit transferred")
```
- bool fields usually use a multi-word form (e.g. `enable_batch_size` or `show_lot_ids`) it says what it does better than just 'labellable'
- 'labellable' is the British form, use Labelable for the US English, also don't forget upper-case for UI

Reply:
Done